prefect_azure/__init__.py

```diff
@@ -3,6 +3,10 @@
     AzureBlobStorageCredentials,
     AzureCosmosDbCredentials,
     AzureMlCredentials,
+    ACICredentials
 )
+from .aci import ACITask
```

Comment:
Thoughts on renaming the module from `aci` to `container_instance` and the class to `ContainerInstanceJob` (or is that class name too long)?

Reply:
I like it! And I don't think it's too long. I'll make the changes.